### DISCUSSION TITLE
Simultaneous Use of docstore across multiple databases

### DISCUSSION THREAD

**Discussion by bobskigit** [ravendb/ravendb#13220] [Q&A] [answered]
Out app uses multiple databases simultaneously, one to provide writing and reading of logging data and one to provide the standard line of business app stuff. Would it be better to have two docstores or is it ok to make multiple sessions targeting different databases? We have huge transactions volumes, many millions of events per day.

**Comment by ayende** [comment, marked as answer]
It doesn't really matter. Both options should have the same performance / memory costs.
